### ISSUE TITLE
osx-64/openblas-0.2.14-3.tar.bz2 not linked properly

### ISSUE THREAD

**Issue by rmcgibbo** [ContinuumIO/anaconda-issues#608]
I don't think the OpenBLAS library for OS X from `https://repo.continuum.io/pkgs/free/osx-64/openblas-0.2.14-3.tar.bz2` is linked properly.

```
$ wget https://repo.continuum.io/pkgs/free/osx-64/openblas-0.2.14-3.tar.bz2
$ tar -xf openblas-0.2.14-3.tar.bz2
$ otool -L lib/libopenblas.dylib
lib/libopenblas.dylib:
    libopenblasp-r0.2.14.dylib (compatibility version 0.0.0, current version 0.0.0)
    /usr/local/lib/libgfortran.2.dylib (compatibility version 3.0.0, current version 3.0.0)
    /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 159.1.0)
    /usr/lib/libgcc_s.1.dylib (compatibility version 1.0.0, current version 1.0.0)
```

I think it should have an `@rpath` entry, instead of linking directly against anything in /usr/local/lib (I assume that's from homebrew?).

(The symptom is that I get a linker failure to find `/usr/local/lib/libgfortran.2.dylib` when I build a compile a package against this openblas on my machine.)

**Comment by anjos** [comment]
This still happens for openblas-0.2.19-0 on OSX. Is there a plan to fix it?

**Comment by nehaljwani** [comment]
This has been fixed for conda package `openblas` >= `0.2.19-1`
```bash
(root) macbook:Downloads nwani$ otool -L /Users/nwani/m3/pkgs/openblas-0.2.19-1/lib/libopenblas.dylib
/Users/nwani/m3/pkgs/openblas-0.2.19-1/lib/libopenblas.dylib:
	@rpath/libopenblas_nehalemp-r0.2.19.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libgfortran.2.dylib (compatibility version 3.0.0, current version 3.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1197.1.1)
```
